lib/david/app_config.rb

```diff
@@ -3,14 +3,16 @@ class AppConfig < Hash
     DEFAULT_OPTIONS = {
       :Block => true,
       :CBOR => false,
+      :DTLS => false,
       :DefaultFormat => 'application/json',
       :Host => ENV['RACK_ENV'] == 'development' ? '::1' : '::',
       :Log => nil,
       :MinimalMapping => false,
       :Multicast => true,
       :MulticastGroups => ['ff02::fd', 'ff05::fd'],
       :Observe => true,
-      :Port => ::CoAP::PORT
+      :Port => ::CoAP::PORT,
+      :PortDTLS => 5684 # TODO
```

Comment:
The CoAP gem should support a `::CoAP::PORT_SECURE` (or something like that) constant, which it doesn't currently, and that constant should be used instead here.

Reply:
See: https://github.com/nning/coap/pull/4